docs/src/components/InteractivePlayground/index.tsx

```diff
@@ -77,7 +100,17 @@ export default function InteractivePlayground(
             />
           </div>
           <div className={styles.previewWrapper}>
-            <React.Fragment key={key}>{example}</React.Fragment>
+            <React.Fragment key={key}>
+              {width !== null &&
+                ((memo ?? []).length > 0 ? (
+                  useMemo(
+                    () => <Example {...exampleProps} width={width} />,
+                    [...memo, width]
+                  )
+                ) : (
+                  <Example {...exampleProps} width={width} />
```

Comment:
Yes, Sir! This memo is used, because in some usePlayground it was used and I needed to have more control over the props, so I decided to move it into InteractivePlayground.